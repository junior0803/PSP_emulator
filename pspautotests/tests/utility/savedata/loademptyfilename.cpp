#include "emptyfilename.h"

extern "C" int main(int argc, char **argv) {
	initDisplay();

	runEmptyFileNameTrials("auto load:", SCE_UTILITY_SAVEDATA_TYPE_AUTOLOAD);
	runEmptyFileNameTrials("load:", SCE_UTILITY_SAVEDATA_TYPE_LOAD);
	runEmptyFileNameTrials("list load:", SCE_UTILITY_SAVEDATA_TYPE_LISTLOAD);
	runEmptyFileNameTrials("read data secure:", SCE_UTILITY_SAVEDATA_TYPE_READDATASECURE);
	runEmptyFileNameTrials("read data:", SCE_UTILITY_SAVEDATA_TYPE_READDATA);

	return 0;
}