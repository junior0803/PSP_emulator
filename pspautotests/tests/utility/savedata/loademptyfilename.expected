[x] auto load:
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:0): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] Secure:TEST1.OBJ
[x] System:PARAM.SFO
[x] loaddata:Orignal data

[x] load:
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:2): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] Secure:TEST1.OBJ
[x] System:PARAM.SFO
[x] loaddata:Orignal data

[x] list load:
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:4): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] Secure:TEST1.OBJ
[x] System:PARAM.SFO
[x] loaddata:Orignal data

[x] read data secure:
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:15): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] Secure:TEST1.OBJ
[x] System:PARAM.SFO
[x] loaddata:Orignal data

[x] read data:
[r] sceUtilitySavedataInitStart(mode:13): 00000000
[r] sceUtilitySavedataInitStart(mode:16): 00000000
[r] sceUtilitySavedataInitStart(mode:12): 00000000
[r] Files:
[x] Secure:TEST1.OBJ
[x] System:PARAM.SFO
[x] loaddata:Orignal data
